rpcs3/rpcs3qt/save_manager_dialog.cpp

```diff
@@ -510,16 +510,10 @@ void save_manager_dialog::UpdateDetails()
 		m_details_details->setText("");
 		m_details_note->setText("");
 
-		if (selected > 1)
-		{
-			m_button_delete->setDisabled(false);
-			m_details_title->setText(tr("%1 items selected").arg(selected));
-		}
-		else
-		{
-			m_button_delete->setDisabled(true);
-			m_details_title->setText(tr("Select an item to view details"));
-		}
+		m_button_delete->setDisabled( !(selected > 1) );
+		m_details_title->setText(
+			selected > 1 ? tr("%n item(s) selected", "", selected) : tr("Select an item to view details")
+		)
```

Comment:
All of this is unnecessary, because this code is unreachable for n == 1.

Reply:
I wonder why this "if else" was here : any ideas ?

Reply:
for 0 of course

Reply:
the ternary just makes it more difficult to read in this case and adds nothing to readability

Reply:
Feel free to revert that part if you want : I used ternary when I got the opportunity